ThunderCats/UIAlertController+TCAdditions.m

```diff
@@ -43,4 +43,61 @@ + (instancetype)tc_notImplementedAlert
     return alert;
 }
 
++ (instancetype)tc_alertWithTitle:(NSString * __tc_nonnull)title message:(NSString * __tc_nonnull)message cancelButtonTitle:(NSString * __tc_nonnull)cancelButtonTitle cancelAction:(void (^ __tc_nullable)(UIAlertAction * __tc_nonnull))cancelAction
```

Comment:
There's a lot of code duplication between this method and the action sheet method.  We can probably implement a private method that handles all of the common code and simply passes the alert controller style as an argument.